src/Protocol/Diaspora.php

```diff
@@ -2609,23 +2616,34 @@ private static function receiveRetraction(array $importer, $sender, $data)
 	/**
 	 * Checks if an incoming message is wanted
 	 *
-	 * @param string $url
-	 * @param integer $uriid
+	 * @param array  $item
 	 * @param string $author
 	 * @param string $body
+	 * @param int    $direction Indicates if the message had been fetched or pushed
```

Comment:
Please add a reference to the constants the parameter is taking its values from.